SECURITY.md

```diff
@@ -0,0 +1,126 @@
+# Security Policy
+
+## Introduction
+
+Codenotary, September 7th 2022
+
+*This security policy is based on [this template](https://www.cisa.gov/vulnerability-disclosure-policy-template).*
+
+Codenotary is committed to ensuring the security of the American and global public by protecting their information. This policy is intended to give security researchers clear guidelines for conducting vulnerability discovery activities and to convey our preferences in how to submit discovered vulnerabilities to us.
+
+This policy is addressed to people using the immudb and / or contributing to the project. It describes **what systems and types of research** are covered under this policy, **how to send** us vulnerability reports, and **how long** we ask security researchers to wait before publicly disclosing vulnerabilities.
+
+We encourage you to contact us to report potential vulnerabilities in our systems.
+
+## Authorization
+
+If you make a good faith effort to comply with this policy during your security research, we will consider your research to be authorized we will work with you to understand and resolve the issue quickly, and Codenotary will not recommend or pursue legal action related to your research. Should legal action be initiated by a third party against you for activities that were conducted in accordance with this policy, we will make this authorization known.

```

Comment:
> ,we will consider your research to be authorized we will work with you to understand and resolve the issue quickly,

I have a problem understanding if this is grammatically correct, the same is in the template but maybe that's some template issue ?

```suggestion
If you make a good faith effort to comply with this policy during your security research, we will consider your research to be authorized, we will work with you to understand and resolve the issue quickly, and Codenotary will not recommend or pursue legal action related to your research. Should legal action be initiated by a third party against you for activities that were conducted in accordance with this policy, we will make this authorization known.
```

Reply:
I guess, there is a comma missing, after adding it it seems to be correct to me.